### ISSUE TITLE
How to set all admin routes under /admin?

### ISSUE THREAD

**Issue by igorsantos07** [LaravelDaily/quickadmin#136]
Currently, the routes generated by the package are all under the main path, but, ideally, all Admin routes should be under /admin, such as /admin/roles, /admin/users, etc. This way, the root routes are free to be used by the main website.

**Comment by PovilasKorop** [comment]
@igorsantos07 we've solved that in our online generator version - https://quickadminpanel.com

Package version now releases only sever issues fixes and is updated for newer Laravel versions, so we're not going to change the routes. 

But we're accepting PRs if you wish to spend time on this, we would appreciate!

**Comment by igorsantos07** [comment]
Oh.

So the package is sort of abandoned (in maintenance mode, so to speak) and
new features will be sort of paid (since the online generator free version
is quite limited)?

Sad to hear but I can understand how you came to this decision.

Sad there's no other free scaffold for such a great framework as Laravel.
Long live to Django, it seems.

**Comment by LaravelDaily** [comment]
Yes, we're focused on premium online version for couple of years now, and it's successful with a lot of customers, that's why we can reinvest the money and build more features. With free package it is almost impossible to grow without any funding.

And I disagree about lack of scaffolding, see my article on Laravel news. https://laravel-news.com/13-laravel-admin-panel-generators

I personally recommend Voyager, but look at hundreds of issues on their github and you will understand that free package is not a good model for both company and users.

**Comment by igorsantos07** [comment]
I surely understand it's not easy to keep a free package running. I'm myself an abandonware maintainer.

That said, it's sad to see a package with more than 400 stars getting abandoned like that, being it the source of your current funding. If the premium version is that successful, why couldn't it bring some life back into its original package as well?
Maybe you should recommend other packages in the readme, as you're doing here. This way, new users would be aware of better alternatives, besides the paid version of this.

At least you're still accepting PRs :)

**Comment by PovilasKorop** [comment]
@igorsantos07 customers who pay us money don't ask us to reinvest them into a free package, they ask us for more feature in a premium version. And I don't think it's fair to let down people who paid us in favor of people who don't respect our work enough to pay for our work premium version.

This package is very far from being the source of our current funding, it was more like internal experiment at the time, just released in public.

**Comment by igorsantos07** [comment]
But those are other services, just like your premium version is.
Laravel is still free of charge. From your words, if you use Laravel for free, you're unrespectful of him.

I think I sort of got what you mentioned originally, just wanted to show you how ill worded that was.

**Comment by PovilasKorop** [comment]
@igorsantos07 maybe I've phrased it wrong, sorry.

My point is that open-source people need to earn money one way or another, and we've found our way. And now we're doing much more in open-source - see here https://github.com/LaravelDaily - whereas this package is not maintained actively for a while now. Probably I should update readme.

But it makes me sad that people are complaining that they are not getting something for free. That's why I'm saying "unrespectful". It's like receiving an iPhone for a gift, and then complaining that you don't get 5-years warranty with it.
